enterprise_subsidy/apps/api_client/enterprise.py

```diff
@@ -28,19 +28,32 @@ class EnterpriseApiClient(BaseOAuthClient):
     """
     api_base_url = settings.LMS_URL + '/enterprise/api/v1/'
     enterprise_customer_endpoint = api_base_url + 'enterprise-customer/'
+    enterprise_subsidy_fulfillment_endpoint = api_base_url + 'enterprise-subsidy-fulfillment/'
 
     def enterprise_customer_url(self, enterprise_customer_uuid):
         return os.path.join(
             self.enterprise_customer_endpoint,
             f"{enterprise_customer_uuid}/",
         )
 
+    def enterprise_fulfillment_url(self, enterprise_fulfillment_uuid):
+        return os.path.join(
+            self.enterprise_subsidy_fulfillment_endpoint,
+            f"{enterprise_fulfillment_uuid}/"
+        )
+
     def enterprise_customer_bulk_enrollment_url(self, enterprise_customer_uuid):
         return os.path.join(
             self.enterprise_customer_url(enterprise_customer_uuid),
             "enroll_learners_in_courses/",
         )
 
+    def enterprise_fulfillment_cancel_url(self, enterprise_fulfillment_uuid):
+        return os.path.join(
+            self.enterprise_fulfillment_url(enterprise_fulfillment_uuid),
+            "cancel-enrollment",
```

Comment:
is this not called "cancel-fulfillment"? https://github.com/edx/edx-enterprise/blob/f9ed55cad034b3c15c3a47c949cc8937a6f4ca0b/enterprise/api/v1/urls.py#L56

Reply:
whoops!